project-board-crew-logic/src/main/java/project/board/crew/logic/structure/categories/CrewCategoriesService.java

```diff
@@ -0,0 +1,21 @@
+package project.board.crew.logic.structure.categories;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CrewCategoriesService {
+
+    private final CrewCategoriesRepository crewCategoriesRepository;
+
+    public CrewCategoriesService(CrewCategoriesRepository crewCategoriesRepository) {
+        this.crewCategoriesRepository = crewCategoriesRepository;
+    }
+
+    public CrewCategories add(CrewCategories crewCategories)
+    {
+        if (crewCategoriesRepository.existsById(crewCategories.getCategoryId()))
```

Comment:
To samo co w Crew, jak będziesz miał generowane Id to nie będziesz potrzebował sprawdzać czy jest poprawne

Reply:
++++